README.md

```diff
@@ -0,0 +1,43 @@
+Montréal-Python Code of Conduct
+===============================
+
+Presentation
+------------
+
+This is a "fork" of the PSF code of conduct and this README was inspired by the one of the PyCon code of conduct.
+
+Thank you to Adria Richards and many others with the Python Software Foundation and Python community for recommending that we publicly discuss and version control these documents so changes can be suggested, wording corrected and we can continue to iterate, as a community on these policies and procedures.
+
+Thanks to the members of the Montréal-Python user group for taking time to translate the code of conduct of the PSF in French.
+
+Montréal-Python and the community as a whole are still very much in a learning process, the group is 10 years old and still evolving daily. Each year we see massive improvements (such as the code of conduct).
+
+We hope, as a team we can iterate and improve continually.
+
+No changes are final until approved by the President of the Montréal-Python user group and versions in place for a given year will be clearly marked, the website contents on montrealpython.org will clearly state the revision of the documents posted.
+
+You can have access to our current code of conduct by clicking [here](https://github.com/mtlpy/code-of-conduct/blob/master/CodeOfConduct.rst).
```

Comment:
C’est mieux pour plusieurs raisons (métadonnées, accessibilité, etc) de ne pas utiliser «click here» comme texte de liens. Proposition:

`Follow this link to read the text: [Montréal-Python code of conduct](CodeOfConduct.rst)`